server/flaskr/database/postgres/handlers/organization_data_handler.py

```
@@ -0,0 +1,91 @@
+            result = cur.fetchone()
+        return result[0] if result else None
+    
+
+    @classmethod
+    def add_invite(cls, orgId, email):
+        query = "INSERT INTO public.invitations (orgId, email) VALUES (%s, %s);"
+        params = (orgId, email)
+        write_query(query, params)
+    
+    @classmethod
+    def delete_invite(cls, email):
```

Ruff (0.15.15)

[warning] Missing return type annotation for classmethod `delete_invite`

Add return type annotation: `None`

(ANN206)